proto/osmosis/claim/v1beta1/genesis.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+package osmosis.claim.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/bank/v1beta1/genesis.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/c-osmosis/osmosis/x/claim/types";
+
+enum Action {
+	option (gogoproto.goproto_enum_prefix) = false;
+  
+	ActionAddLiquidity = 0;
+	ActionSwap = 1;
+	ActionVote = 2;
+	ActionDelegateStake = 3;
+}
+
+message UserActivities {
+  string user = 1;
+  repeated Action actions = 2;
+}
+
+message UserActivity {
+  string user = 1;
+  Action action = 2;
+}
+
+// GenesisState defines the claim module's genesis state.
+message GenesisState {
```

Comment:
Can we have some comment about the structure of whats here?

In particular, I'm unclear on how the `address -> claimed activities` is stored, and what `claimables` is

Reply:
Is user activities going to be all remaining claimable funds, stored as a list of 
```
(account_string, [Action1 coins remaining, Action2 coins remaining, ...])
```
?

Reply:
```
  // the maximum claimable amount per address
  // [(account_string, max_claimable), (account_string, max_claimable) ... ]
  repeated cosmos.bank.v1beta1.Balance claimables = 5
    [(gogoproto.nullable) = false];
  // the actions done per address
  // [(account_string, [done_action1, done_action2]), (account_string, [done_action1]) ... ]
  repeated UserActivities activities = 6[(gogoproto.nullable) = false];
```